gradle/publishing.gradle

```diff
@@ -62,32 +61,9 @@ publishing {
             addPublishingInfo(publication)
         }
     }
-    def publishBuild = Boolean.parseBoolean(findProperty('publishBuild') ?: 'false')
-    if (publishBuild && System.getenv('BINTRAY_USER') != null && System.getenv('BINTRAY_KEY') != null) {
-        def isReleaseBuild = Boolean.parseBoolean(findProperty('releaseBuild') ?: 'false')
-        bintray {
-            user = System.getenv('BINTRAY_USER')
-            key = System.getenv('BINTRAY_KEY')
-            publications = ["library"]
-            pkg {
-                version{
-                    name = project.version
-                    mavenCentralSync {
-                        sync = false
-                    }
-                }
-                userOrg = "fdb-apple"
-                repo = "fdb-record-layer"
-                name = project.name
-                licenses = ["Apache-2.0"]
-                vcsUrl = "https://github.com/FoundationDB/fdb-record-layer.git"
-            }
-            override = !isReleaseBuild
-            publish = publishBuild
-        }
-    }
 
     artifactoryPublish {
+        def publishBuild = Boolean.parseBoolean(findProperty('publishBuild') ?: 'false')
         skip = !publishBuild
```

Comment:
what are we actually doing with this? It looks like publications is running whether or not we set it.

Reply:
The idea is that as a safe-guard, one should need to add `-PpublishBuild=true` to the gradle command to actually start sending things to artifactory. How useful this is as a safe-guard is somewhat questionable, I suppose, though I was mostly trying to maintain compatibility with the old system unless there was a reason to change it (`mutatis mutandis` and all that).

That being said, there also seems to be a bug with skip, I think. It's supposed to not run the publishing (see: https://gist.github.com/DALDEI/40925568cef57e16e8a7), so I added an an additional condition

Reply:
thanks!